content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -147,6 +147,49 @@ The RUM SDK automatically monitors frameworks that rely on hash (`#`) navigation
 
 ## Add your own performance timing
 
+### Custom Vitals
+
+The `customVital` API allows you to measure the performance of your application at the component or the user flow level. 
+
+#### startDurationVital and stopDurationVital
+
+You can start a duration measurement by calling `startDurationVital` and stopping it later in your code with `stopDurationVital`:
+
+```javascript
+window.DD_RUM.startDurationVital("dropdownRendering")
+window.DD_RUM.stopDurationVital("dropdownRendering")
+```
+
+Once you call the `stopDurationVital` method, the custom vital duration is sent to Datadog and can be queried using `@vital.name:dropdownRendering`. For example, you can filter by duration with `@vital.duration:<10`.
```

Comment:
let's use '@vital.duration:>10' to demonstrate how to filter for long durations.